katalog/velero/velero-aws/README.md

```diff
@@ -25,19 +25,18 @@ by this deployment including the Cloud Credentials file.
 
 ```bash
 $ terraform init
+# omitted output
 $ terraform apply
-$ terraform output cloud_credentials > /tmp/cloud_credentials.conf
-$ cat /tmp/cloud_credentials.conf
-[default]
-aws_access_key_id=AKIAEXAMPLE
-aws_secret_access_key=aosidaoidhqwd09uuqwda-asd
+# omitted output
+$ terraform output cloud_credentials > /tmp/cloud_credentials.yaml
+# omitted output
 ```
 
 Then you are ready to apply this file in the `kube-system` namespace:
 
 ```bash
-$ kubectl create secret generic cloud-credentials --from-file=cloud=/tmp/cloud_credentials.conf --dry-run -o yaml | kubectl apply -f - -n kube-system
-# omitted output
+$ kubectl apply -f /tmp/cloud_credentials.yaml -n kube-system
+secret/cloud-credentials created
```

Comment:
This should probably be updated according to the result of the discussion above

Reply:
As it is not a huge change, I prefer not to maintain both outputs.
We are going (probably) to add a new output/input to support serviceAccounts in AWS.

I want to keep it as simple and clean as possible. 

If there is something blocking from the delivery team, don't doubt to expose it :)

Reply:
Nothing blocking, but I think it's important to highlight that this is a breaking change that will require manual intervention since it breaks kustomize's SecretGenerator usage.

Reply:
It is kind of mentioned in the release notes: https://github.com/sighupio/fury-kubernetes-dr/blob/v1.5.0-rc3/docs/releases/v1.5.0.md#terraform-modules